### ISSUE TITLE
arm64 support

### ISSUE THREAD

**Issue by rust0258** [kubevious/helm#29]
Hi,
I am trying to install Kubevious with helm chart on EKS with arm64 node arch, I think Kubevious don't have arm support yet.
I am getting this error from the backend pod logs:
```
$ kubectl logs kubevious-backend-54b67c94-qcr62 -n kubevious
exec /usr/local/bin/docker-entrypoint.sh: exec format error
```

**Comment by rubenhak** [comment]
@ibrahimjelliti, yes, images are not built for arm64. If you can help with building them for arm64 I can incorporate them into the helm chart. 

Here is the [Dockerfile](https://github.com/kubevious/backend/blob/main/Dockerfile) and the [CI build step](https://github.com/kubevious/backend/blob/7738b254ad16faeb0d5609b8c4fe5c2d02d9c419/.codefresh/build.yml#L142).

Using CodeFresh for CI/CD. Looks like they have buildx/buildkit support. Not sure whichc settings are right to use. Here are the docs: 
https://codefresh.io/docs/docs/pipelines/steps/build/
